sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -120,36 +155,53 @@ class CorsRule(GeneratedCorsRule):
 
     All required parameters must be populated in order to send to Azure.
 
-    :param list(str) allowed_origins:
+    :param List[str] allowed_origins:
         A list of origin domains that will be allowed via CORS, or "*" to allow
-        all domains. The list of must contain at least one entry. Limited to 64
+        all domains. The list must contain at least one entry. Limited to 64
         origin domains. Each allowed origin can have up to 256 characters.
-    :param list(str) allowed_methods:
+    :param List[str] allowed_methods:
         A list of HTTP methods that are allowed to be executed by the origin.
-        The list of must contain at least one entry. For Azure Storage,
+        The list must contain at least one entry. For Azure Storage,
         permitted methods are DELETE, GET, HEAD, MERGE, POST, OPTIONS or PUT.
     :keyword int max_age_in_seconds:
         The number of seconds that the client/browser should cache a
         pre-flight response.
-    :keyword list(str) exposed_headers:
+    :keyword List[str] exposed_headers:
         Defaults to an empty list. A list of response headers to expose to CORS
         clients. Limited to 64 defined headers and two prefixed headers. Each
         header can be up to 256 characters.
-    :keyword list(str) allowed_headers:
+    :keyword List[str] allowed_headers:
         Defaults to an empty list. A list of headers allowed to be part of
         the cross-origin request. Limited to 64 defined headers and 2 prefixed
         headers. Each header can be up to 256 characters.
     """
 
-    def __init__(self, allowed_origins, allowed_methods, **kwargs):
+    allowed_origins: str
+    """The comma-delimited string representation of the list of origin domains that will be allowed via
+        CORS, or "*" to allow all domains. The list must contain at least one entry. Limited to 64
+        origin domains. Each allowed origin can have up to 256 characters."""
```

Comment:
Going to point this out specifically here -- but feedback will apply to the whole file.

Do we want to include ALL the information from the docstring? I understand giving a blurb that describes it, but for example I think here maybe we could consider only having this:
```
"""The comma-delimited string representation of the list of origin domains that will be allowed via
        CORS, or "*" to allow all domains."""
```
since the full nitty gritty details exist in the docstring above. However for things like ivar, I would leave ALL the information since we would effectively remove the ivar docstrings.